spec.md

```diff
@@ -25,6 +25,8 @@
     - [Types of Binding](#types-of-binding)
   - [Unbinding](#unbinding)
   - [Deprovisioning](#deprovisioning)
+  - [Fetching a Service Instance](#fetching-a-service-instance)
```

Comment:
Minor thing but the order of the TOC seems to be: provision, update instance, bind, unbind, deprovision - which is the normal lifecycle of things. I would think "fetching" should be in the middle of that flow, not at the end.